cdap-watchdog/src/main/java/io/cdap/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -161,8 +162,7 @@ public class DefaultMetricStore implements MetricStore {
     aggs.put(BY_WORKFLOW, new DefaultAggregation(
         ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
             Constants.Metrics.Tag.WORKFLOW, Constants.Metrics.Tag.DATASET,
-            Constants.Metrics.Tag.RUN_ID, Constants.Metrics.Tag.NODE,
-            Constants.Metrics.Tag.APP_ENTITY_TYPE, Constants.Metrics.Tag.APP_ENTITY_TYPE_NAME),
+            Constants.Metrics.Tag.RUN_ID, Constants.Metrics.Tag.NODE),
```

Comment:
I'd separate this PR into two: one as the revert commit in https://github.com/cdapio/cdap/pull/15170, the other focuses on adding the metric aggregation.

Reply:
Sure, once the other PR is merged, I'll rebase this branch and force-push